members/templates/members/union_members.html

```diff
@@ -0,0 +1,101 @@
+{% extends 'members/base.html' %}
+
+{% block content %}
+  <div id="union_info">
+      <h1>Foreningen for {{union.name}}</h1>
+      <strong>Adresse: </strong>{{union.address}}
+      <div>
+        <h4>Bestyrelse</h4>
+        <div>
+          <div>
+              <ul>
+                <li>
+                    <strong>Formand:</strong>
+                    <a href="mailto:{{union.chairman_email}}">{{union.chairman}}</a>
+                </li>
+                <li>
+                    <strong>Næstformand:</strong>
+                    <a href="mailto:{{union.second_chair_email}}">{{union.second_chair}}</a>
+                </li>
+                <li>
+                    <strong>Kasserer:</strong>
+                    <a href="mailto:{{union.cashier_email}}">{{union.cashier}}</a>
+                </li>
+                <li>
+                    <strong>Sekretær:</strong>
+                    <a href="mailto:{{union.secratary_email}}">{{union.secretary}}</a>
+                </li>
+                <li>
+                    <strong>Foreningsmail:</strong>
+                    <a href="mailto:{{union.union_email}}">{{union.union_email}}</a>
+                </li>
+              </ul>
+          </div>
+          <div>
+            <strong>Menige medlemmer</strong>
+            <div>
+              {{union.boardMembers|linebreaks}}
+              <!--{% for boardMember in union.boardMembers %}
+                <a href="mailto:{{boardMember.email}}">{{boardMember.name}}</a><br>
+              {% endfor %}-->
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div id="union_documents">
+        {% if union.statues %}
+            <a href="{{union.statues}}"><strong>Vedtægter</strong></a>
+        {% endif %}
+        {% if union.meeting_notes %}
+            <a href="{{union.meeting_notes}}"><strong>Referater</strong></a>
+        {% endif %}
+    </div>
+    {% if access == True%}
+      <ul class="nav nav-pills nav-fill">
```

Comment:
I stedet for at bruge navpills/tab-content fra bootstrap så brug den der `tabs` fra vores egen styling så det passer med designet på resten af siden. 
https://github.com/CodingPirates/forenings_medlemmer/wiki/HTML-formatting